catalog/core/catalog-core-camelcomponent/src/test/java/ddf/camel/component/catalog/CatalogComponentFrameworkTest.java

```diff
@@ -423,7 +423,7 @@ public void testCreateWithInvalidOperation() throws Exception {
     mockVerifierEndpoint.assertIsSatisfied();
   }
 
-  @Test
+  @Test(expected = CamelExecutionException.class)
```

Comment:
❓Since we're actually expecting a different exception than the one thrown (`IngestException`) in these tests, would it be worth using JUnit's https://junit.org/junit4/javadoc/latest/org/junit/rules/ExpectedException.html and validate that the cause is what we expect?